tests/playwright/README.md

```diff
@@ -0,0 +1,14 @@
+# Playwright Testing
+
+## Local Environment Variables
+Allows using environment variables inside the tests.
+
+- Duplicate `.env.sample` and rename it as `.env`, set values and add new variables if needed.
+
+## Run Tests
+```
+$ npx playwright test
+$ npx playwright test example.spec.js --headed
+$ npx playwright test example.spec.js --debug
+$ npx playwright test -g "Test name here"
```

Comment:
Would be great to add the first one to `package.json` similar to `ddev:e2e-tests` (`ddev:playwright-tests`, ...), and to `ddev:test`.

Reply:
The problem is that the tests that opens PayPal modal does not work without opening the browser (`--headed`), the idea is to create some basic tests that calls the PayPal API instead of opening the browser (https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests) and use only these for CI.

 I would rather prefer to include those in package.json once implemented, for now the idea is just have Playwright available to help us in development by reducing manual testing.

Reply:
It's not for CI, CI executes the commands directly. It's to avoid having to remember/copy these commands every time during development, or installing npx on host (or executing `ddev ssh` before).

And if `--headed` is needed, then it probably should filter them in some simpler way than adding file names one by one, maybe there is a way to define groups like in PHPUnit and then execute the whole headed or non-headed group.